examples/resources/genesyscloud_guide_version/resource.tf

```diff
@@ -0,0 +1,23 @@
+resource "genesyscloud_guide_version" "sample-guide" {
+  guide_id    = data.genesyscloud_guide.id
+  instruction = "This is a test Instruction"
+  variables {
+    name        = "TestVariable"
+    type        = "String"
+    scope       = "InputAndOutput"
+    description = "This is a test Description"
+  }
+  resources {
+    data_action {
+      data_action_id = "DataActionId"
```

Comment:
Is this supposed to be a reference to an actual GC Data Action? If so, you should pull in those references from existing examples.